trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/GravitinoConnectorPluginManager.java

```diff
@@ -141,40 +155,40 @@ private void loadPlugin(String pluginPath, String pluginName) {
                   "org.openjdk.jol.",
                   "io.opentelemetry.api.",
                   "io.opentelemetry.context."));
-      pluginClassLoaders.put(pluginName, (ClassLoader) pluginClassLoader);
+
+      ServiceLoader<Plugin> serviceLoader =
```

Comment:
Why move this logic here? It's in the `createConnector` before.

Reply:
Previously, the class only supported loading plugins from a directory, but in the test environment, we don't have a plugin directory. However, we can create a plugin immediately. Therefore, we have added a new interface, `installPlugin(name, plugin)`, and then refactored the code to move the plugin loading from the `createConnector ` function.